Source/ACE/Network/GameAction/Actions/GameActionMoveToState.cs

```diff
@@ -90,7 +90,9 @@ public static void Handle(ClientMessage message, Session session)
             forcePositionTimestamp = message.Payload.ReadUInt16();
             message.Payload.ReadByte();
 
-            session.Player.UpdateLocation(position);
+            // FIXME(ddevec): The game sends invalid positions sometimes... when in portal space.  Ignore?
+            if (position.Cell != 0)
```

Comment:
I'm thinking we might ought to have a Position constant that indicates portal space.  This way you don't leave the character behind somewhere too long where they can still be targeted by monsters or something after they've portaled away.